src/app/admin/admin.service.ts

```diff
@@ -34,6 +34,11 @@ export class AdminService {
 
   private listAllConfigVarialbeUrl = environment.backendAppUrl + "/api/admin/config/fetch/list";
 
+  private refreshingUserLocationCache = environment.backendAppUrl + "/api/user/profile/user/location/refresh";
+  private fetchUsersWithLocation = environment.backendAppUrl + "/api/group/fetch/users/location/timeStamp/";
```

Comment:
Make sure to match the URL convention of the other lines. Having a "/" or not "/" can break systems.